Dockerfile

```
@@ -1,6 +1,6 @@
 FROM ocaml/opam:ubuntu-22.04-ocaml-4.14@sha256:cdc3ec85f42f2daaea17db5024405a9f8f0c83766d35626078eb0c33aa1a8b53 AS build
```

```suggestion
FROM ocaml/opam:ubuntu-22.04-ocaml-4.14@sha256:1e42940a3a666fe90409091218445fb763fc424d356815d291f4daa44f0db54f AS build
```

This image doesn't package `git-unix.3.12` that you want, but it's the most recent one.